projects/lights_control/src/lights_gpio.c

```diff
@@ -62,6 +63,7 @@ StatusCode lights_gpio_process_event(const LightsGpio *lights_gpio, const Event
   if ((e->id != LIGHTS_EVENT_GPIO_OFF) && (e->id != LIGHTS_EVENT_GPIO_ON)) {
     return STATUS_CODE_OK;
   }
+  // LOG_DEBUG("Turning %s: %d\n", (e->id == LIGHTS_EVENT_GPIO_ON) ? "ON" : "OFF", e->data);
```

Comment:
Please remove commented out code

Reply:
Can I uncomment it and keep it? It's pretty useful for debugging stuff.